### PR TITLE
Add 'member' to getProfile state output, and add some dev-env mock data

### PR DESCRIPTION
**Pull request by pfrazee**
- Adds `member` to the `myState` output of `getProfile`. The value is the URI of the confirmation record.
- Adds some scene mock data to the dev environment

### CODE REVIEW

File: packages/dev-env/src/mock/index.ts [hunk 1]
```diff
@@ -121,6 +121,65 @@ export async function generateMockSetup(env: DevEnv) {
...
+      {
+        assertion: APP_BSKY_GRAPH.AssertMember,
+        subject: {
+          did: bob.did,
+          declarationCid: bob.declarationCid,
+        },
+        createdAt: new Date().toISOString(),
+      },
+    )
+    bob.api.app.bsky.graph.confirmation.create(
```
Comment (devinivy) [inline packages/dev-env/src/mock/index.ts:140]: I think we want these `await`s, but feel free to ignore if it was intentional as-is.
```suggestion
    await bob.api.app.bsky.graph.confirmation.create(
```

File: packages/dev-env/src/mock/index.ts [hunk 2]
```diff
@@ -121,6 +121,65 @@ export async function generateMockSetup(env: DevEnv) {
...
+      {
+        assertion: APP_BSKY_GRAPH.AssertMember,
+        subject: {
+          did: carla.did,
+          declarationCid: carla.declarationCid,
+        },
+        createdAt: new Date().toISOString(),
+      },
+    )
+    carla.api.app.bsky.graph.confirmation.create(
```
Comment (devinivy) [inline packages/dev-env/src/mock/index.ts:167]:
```suggestion
    await carla.api.app.bsky.graph.confirmation.create(
```

**Review by devinivy — APPROVED** [review]
Excellent!

**Comment by pfrazee** [comment]
Got it, thanks for the tweaks. (Holy cow that's a useful github feature)

**Pull request merged**